README.md

```diff
@@ -69,4 +69,56 @@ bundle exec rspec
 
 ### Dev tips
 
-- To get Rubocop to fix detected issues automatically (where it can): `bundle exec rubocop -a`
+- To get Rubocop to fix detected issues automatically (where it can):
+  - `bundle exec rubocop -a`
+
+## Testing subscription emails
+
+### Local dev
+
+#### Email templates
+
+When working on email templates, [Action Mailer Previews](https://guides.rubyonrails.org/action_mailer_basics.html#previewing-emails) provide a way to test the output of these templates without having to actually send any emails. We have one set up for subscription lesson emails – example usage:
+
+http://localhost:3000/rails/mailers/lesson_mailer/lesson_email?course=how-to-manage-your-money&lesson_no=1&languages=en,ar&disguised=true
+
+Note: this preview currently uses the first `User` in the database, so make sure you have one registered locally!
+
+#### Full subscription email flow
+
+You can also test out the full subscription email flow locally. We use [mailcatcher](https://github.com/sj26/mailcatcher) to capture all sent email from the local app. An instance of mailcatcher will already be running if you've used the Docker Compose set up mentioned above – you can open this at http://localhost:1080/.
+
+Steps to test:
+
+1. Start up the Sidekiq background worker using:
+  - `bundle exec sidekiq -c 1`
+2. Make or edit a new subscription in the app – make sure the schedule is set accordingly (i.e. to send you an email within the current time slot).
```

Comment:
Add in note about being aware of the current timezone you are in

Reply:
Good point! Will add in.